libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/AdvancedNetworkFields.tsx

```diff
@@ -37,57 +37,80 @@ const AdvancedNetworkFields = () => {
 
   const isDualStack = values.stackType === DUAL_STACK;
 
+  // Keep latest arrays in refs so we can avoid re-running effect on every array edit
+  const clusterNetworksRef = React.useRef(values.clusterNetworks);
+  const serviceNetworksRef = React.useRef(values.serviceNetworks);
+  React.useEffect(() => {
+    clusterNetworksRef.current = values.clusterNetworks;
+  }, [values.clusterNetworks]);
+  React.useEffect(() => {
+    serviceNetworksRef.current = values.serviceNetworks;
+  }, [values.serviceNetworks]);
```

Comment:
You dont need refs actually. You can keep using the `values.cluster/serviceNetworks` in useEffect, but leave them out of the dependency array.
Creating refs just to keep the eslint validation happy is not the correct way.